### ISSUE TITLE
Rule: Detect same variables

### ISSUE THREAD

**Issue by mnojek** [MarketSquare/robotframework-robocop#120]
RF ignores case, spaces and underscore in variable names and it's sometimes confused that
```
${some var}
${someVar}
${some_var}
${SOME_VAR}
```
are treated like the same variable. This can lead to bugs when someone overwrites the variable using very similar but different name.
It should be reported by robocop to avoid potential bugs.

**Comment by mnojek** [comment]
This ticket is now 1 years old! I think we can wish it all the best and fast way to be closed without errors and regression! 🎆 🍾 💐

**Comment by bhirsz** [comment]
So that linked PR is your idea for a great birthday gift  🎁 ;)

**Comment by mnojek** [comment]
There are still 3 other issues from 2020. Hopefully, we can close them soon before 2022 😀 Don't let them age!
